chart/openfaas/README.md

```diff
@@ -48,6 +48,22 @@ kubectl -n openfaas create secret generic basic-auth \
 --from-literal=basic-auth-password="$PASSWORD"
 ```
 
+Generate secrets so that we can enable non-repudiation for asynchronous function callbacks:
```

Comment:
I think this should be moved to the appendix as an optional (rather than default feature)

Having `faas-cli install` later down the line will help abstract/automate this additional work.

Reply:
@alexellis 👍 moved to bottom